sklearn/decomposition/pca.py

```diff
@@ -353,6 +353,16 @@ def fit_transform(self, X, y=None):
         -------
         X_new : array-like, shape (n_samples, n_components)
 
+        Notes
+        -----
+        This method returns an f-ordered array.
+        To get the output array in c-order,
+        do a transpose of the result i.e
+
+        y = PCA().fit_transform(x)
```

Comment:
please indent to show in code font.

Reply:
This still needs addressing. See http://www.sphinx-doc.org/es/stable/markup/code.html#directive-code-block on how to include a font block in docstrings.